.env.example

```diff
@@ -108,6 +108,11 @@ DROPBOX_OAUTH_SECRET=
 WUNDERLIST_OAUTH_KEY=
 WUNDERLIST_OAUTH_SECRET=
 
+EVERNOTE_OAUTH_KEY=
+EVERNOTE_OAUTH_SECRET=
+# Set to true in development, false in production
```

Comment:
I'd actually recommend leaving a comment with your recommended usage, since people will likely not know what the recommended best practice is.